docs/misc/projects.rst

```diff
@@ -122,3 +122,17 @@ Reinforcement learning is used to train agents to control pistons attached to a
 | Email: jackberkowitz88@gmail.com
 | GitHub: https://github.com/jaberkow/WaveRL
 | Presentation: http://bit.ly/WaveRLslides
+
+Fenics-DRL: Fluid mechanics and Deep Reinforcement Learning
+-----------------------------------------------------------
+
+Deep Reinforcement Learning is used to train agents to control the position or the shape of obstacles in different fluids. `Fenics <https://fenicsproject.org>`_ is used for the Fluid Mechanics part, and Stable Baselines is used for the DRL. 
```

Comment:
Maybe add the reason why you want to control position and shape of obstacles.
I think you can also symplify "train agents to control" to "to control"